src/dune_engine/config.mli

```diff
@@ -129,6 +142,7 @@ module type S = sig
     ; cache_duplication : Caching.Duplication.t field
     ; cache_trim_period : int field
     ; cache_trim_size : int64 field
+    ; rpc : Rpc.t field
```

Comment:
I would keep this record for what goes in the configuration file.

Reply:
Hmm, that's a good point. So I suppose we need a separate record for initializing the scheduler. So that we can use this for both `go` and `poll`.

Reply:
Actually, I'm wondering if we should have a `Scheduler.Config.t` type that includes the subset of `Config.t` that is relevant to the scheduler (which would include rpc). It doesn't seem right to me that the scheduler should know about every option that could be defined in the dune-config file, and all of that stuff shouldn't even live in the engine.

Reply:
Seems about right to me

Reply:
Done